2d/benchmarks/dambreak_Colagrossi/test_dambreak_Colagrossi.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+import pytest
+import unittest
+from proteus.iproteus import *
+from proteus import Comm
+comm = Comm.get()
+import dambreak_Colagrossi_so
+import os
+from numpy import *
+from scipy import *
+from pylab import *
+import collections as cll
+import csv
+
+class TestDambreakCollagrossiTetgen(unittest.TestCase):
+
+    @classmethod
+    def setup_class(cls):
+        pass
+
+    @classmethod
+    def teardown_class(cls):
+        pass
+
+    def setup_method(self,method):
+        pass
+
+    def teardown_method(self,method):
+        """ Tear down function """
+        FileList = ['dambreak_Colagrossi.xmf',
+                    'dambreak_Colagrossi.h5']
+        for file in FileList:
+            if os.path.isfile(file):
+                os.remove(file)
+            else:
+                pass
+            
+    def test_run(self):
+        from petsc4py import PETSc
+        pList = []
+        nList = []
+        for (p,n) in dambreak_Colagrossi_so.pnList:
+            pList.append(__import__(p))
+            nList.append(__import__(n))
+            if pList[-1].name == None:
+                pList[-1].name = p
+        so = dambreak_Colagrossi_so
+        so.name = "dambreak_Colagrossi"
+        if so.sList == []:
+            for i in range(len(so.pnList)):
+                s = default_s
+                so.sList.append(s)
+        Profiling.logLevel=7
+        Profiling.verbose=True
+        # PETSc solver configuration
+        OptDB = PETSc.Options()
+        with open("../../../inputTemplates/petsc.options.asm") as f:
```

Comment:
@alistairbntl I  wrote  this bit in order to  add petsc  options from an options file in a script. In addition to the fact  that it might be  useful elsewhere and is  likely going to  get  copy/pasted all  over  the place if we don't find a home in proteus, it's also using a  relative path so the tests will break if  you run `py.test ...` from the root directory like we'll want to to do with travis. On the latter, have you developed a standard practice for making these paths absolute? I remember messing with some approaches but haven't looked at it in a while. I guess one approach is to put these  options in a standard location in the proteus package along with this bit  of code to directly add to PETS.Options.